apps/dsb-client-gateway-api/src/app/types/environment-variables.ts

```diff
@@ -120,6 +120,34 @@ export class EnvironmentVariables {
   @ValidateIf(EnvironmentVariables.isAWSSecretsManagerEnabled)
   AWS_SECRET_PREFIX = '/ddhub/';
```

Comment:
@hejkerooo @nik-ewf
Is this prefix common to any vault type and not just AWS?

Reply:
Azure + Vault have their own prefix too. The vault prefix is for the vault storage name, the other ones are just tags really. We could modify it so there's only one `SECRET_PREFIX` that defaults to `ddhub/`? Thoughts?